.github/workflows/release.yaml

```diff
@@ -39,3 +43,22 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           CR_GENERATE_RELEASE_NOTES: true
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push charts to GHCR
+        run: |
+          shopt -s nullglob
+          for pkg in .cr-release-packages/*; do
+            if [ -z "${pkg:-}" ]; then
+              break
+            fi
+            lowercase=$(echo "${GITHUB_REPOSITORY}" | tr '[:upper:]' '[:lower:]')
```

Comment:
Is this lowercasing necessary? Can we copy prometheus's solution: https://github.com/prometheus-community/helm-charts/blob/1a9a385e3c1cb04b1bcd266df1b334561602e545/.github/workflows/release.yaml#L53-L61

Reply:
It depends on whether you might have a chart with uppercase letters in the future. That's why I included it by default, but I can remove it if you prefer. :)

Reply:
I can't test in my fork as I've uppercase in my github name